test/sphere_diffusion_vec.jl

```diff
@@ -0,0 +1,118 @@
+using Test
+using StaticArrays, IntervalSets
+import ClimaCore.DataLayouts: IJFH
+import ClimaCore:
+    Fields, Domains, Meshes, Topologies, Spaces, Operators, Geometry
+using StaticArrays, IntervalSets, LinearAlgebra
+include("sphere_Plm.jl")
+
+# diffusion on vector: input covariant vecotor
+function ∇²(u)
+    scurl = Operators.Curl()
+    sdiv = Operators.Divergence()
+    wcurl = Operators.WeakCurl()
+    wgrad = Operators.WeakGradient()
+
+    χ = Spaces.weighted_dss!(
+        @. wgrad(sdiv(u)) - Geometry.Covariant12Vector(
+            wcurl(Geometry.Covariant3Vector(scurl(u))),
+        )
+    )
+    return χ
+end
+
+@testset "diffusion on vectors" begin
+    FT = Float64
+
+    l = Int(7)
+    m = Int(4)
+
+    Ne = 7
+    Nq = 6
+
+    radius = FT(1)
+    domain = Domains.SphereDomain(radius)
+    mesh = Meshes.Mesh2D(domain, Meshes.EquiangularSphereWarp(), Ne)
+    grid_topology = Topologies.Grid2DTopology(mesh)
+
+    quad = Spaces.Quadratures.GLL{Nq}()
+    space = Spaces.SpectralElementSpace2D(grid_topology, quad)
+    coords = Fields.coordinate_field(space)
+
+    # compute vector spherical harmonics (VSH) as the gradient of scalar spherical harmonics 
+    # eigenfunction properties of VSH: ∇² = -l(l+1) B⃗ₗᵐ 
```

Comment:
The last symbol shows up as a squiggle to me.

Reply:
@LenkaNovak what browser and OS are you using?

Reply:
Ah, I see boxes in Chrome. Might be my old friend https://bugs.chromium.org/p/chromium/issues/detail?id=1189784

Reply:
I see the right unicode characters in my Chrome on Ubuntu

Reply:
I had this issue with Chrome/Safari on Mac... My solution is FireFox... But I will remove it during the final clean up. I think it is good to keep the minimal use of unicode

BTW, still working on clean up the functions Oswald coded.

Reply:
I've removed those symbols